examples/v1beta1/spot.yaml

```diff
@@ -0,0 +1,30 @@
+# This example will use spot instance type for all
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.